index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="./assets/css/main.css" />
+  <link rel="stylesheet" href="./assets/css/all.min.css" />
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo</title>
+</head>
+<body>
+  <div class="site-wrapper">
+    <div class="content">
+      <h1 class="title">Todo</h1>
+      <ul class="filter-list">
+        <li class="filter-item">All tasks</li>
+        <li class="filter-item">completed</li>
+        <li class="filter-item">uncompleted</li>
+      </ul>
+      <ul class="todo-list">
+        <li class="todo-item-add" >
```

Comment:
I would suggest put this li outside of the ul (list of todo) and change li to div tag.

Reply:
Done.